examples/acados_matlab_octave/linear_mass_spring_model/env.sh

```diff
@@ -55,10 +55,12 @@ echo "ACADOS_INSTALL_DIR=$ACADOS_INSTALL_DIR"
 # export casadi folder and matlab/octave mex folder
 # MATLAB case
 export MATLABPATH=$MATLABPATH:$ACADOS_INSTALL_DIR/external/casadi-matlab/:$ACADOS_INSTALL_DIR/interfaces/acados_matlab_octave/
+export MATLABPATH=$MATLABPATH:$ACADOS_INSTALL_DIR/external/casadi-matlab/:$ACADOS_INSTALL_DIR/interfaces/acados_matlab_octave/acados_template_mex/
```

Comment:
Why this change of folder? Has the entire mex interface been moved inside there?

Reply:
this is already implemented on acados/master, right? https://github.com/acados/acados/tree/master/interfaces/acados_matlab_octave

Reply:
OK, now I get what you mean :)

Reply:
addressed here: ebea92c84f872